src/modules/core/components/MembersList/MembersListItem.tsx

```diff
@@ -72,7 +81,11 @@ const MembersListItem = <U extends AnyUser = AnyUser>(props: Props<U>) => {
       {/* Disable, as `role` is conditional */}
       {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
       <div
-        className={getMainClasses({}, styles, { hasCallbackFn: !!onRowClick })}
+        className={getMainClasses({}, styles, {
+          hasCallbackFn: !!onRowClick,
+          hasReputation: !!showUserReputation,
+          reputationLoaded: !!reputationLoaded,
```

Comment:
Why do you have to negations here as it's already a boolean?

Reply:
It is pretty redundant, I will remove.